packages/plugin/src/theia.v1.63.authentication.d.ts

```diff
@@ -0,0 +1,191 @@
+/********************************************************************************
```

Comment:
Is this file really needed, or can we simply add it to `theia.d.ts` (is there a lot of incompatibilities between 1.53.2 and 1.63.0).